extensions/roles/ledger_roles_config_test.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package roles
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestIsCommitter(t *testing.T) {
+	require := require.New(t)
```

Comment:
@fqutishat you can just do `require.True(t, IsCommitter())` directly - no need to instantiate `require` in all these tests

Reply:
done